### ISSUE TITLE
Use single dollars for mathematics?

### ISSUE THREAD

**Issue by jmhammond** [parpalak/upmath.me#12]
Can the processor render mathematics within standard single-dollar tags? That way when we use the same markdown file with `pandoc`, it properly renders inline vs. display style mathematics?

**Comment by parpalak** [comment]
There are some reasons for that choise. Now the main question is what to do with backward compatibility.

**Comment by jmhammond** [comment]
~~Could you share the reasons? I know that $$ is not canonical LaTeX and is
instead a holdover from TeX. https://tex.stackexchange.com/questions/503/why-is-preferable-to ~~ 

Edit: I don't really need to know the reasons. I'm starting to use upmath integration in Standard Notes and was just curious. But here's an idea that could solve my problem and not break current documents.

To maintain compatibility with the current implementation, could we include
\( to enter inline math and \) to end math mode? This is LaTeX2e spec. For
example,

\( \sum_{n=1}^\infty \frac{1}{n^2} = \frac{\pi^2}{6} \)

Thanks!

**Comment by parpalak** [comment]
Well, I think it's possible, I'll consider this features for next releases. However, I assume Standard Notes uses a fork of the Upmath code, I have nothing in common with this product.

Anyway I'm going to explain the reasons. The core of the Upmath is a service that convert latex equations to SVG images. When an equation is passed, the service does not know if it comes from an inline context or a block context.

Long time ago I used codecogs.com for this purpose. It generates a block equation unless equation starts with `\inline`.  (This is necessary in rare cases, for example, to reduce the height of a fraction added via `\frac`.) Then I've developed my own service but kept the syntax for compatibility.

So the `$\frac{a}{b}$` would be completely equivalent to `$$\inline\frac{a}{b}$$`. And I’ve coded only the latter for simplicity. In addition, the delimiter in form of double dollars will hardly ever occur in the text of a page, so there will be no problems with escaping.
